README.md

```diff
@@ -26,6 +26,8 @@ This plugin only works with embulk >= 0.8.8.
     - **user**: (string, optional)
     - **password**:  (string, optional)
     - **database**:  (string, required)
+    - **tls_enable**: enable/disable tls (boolean, optional, default false)
+    - **tls_insecure**: enable/disable certificate and host validation (boolean, optional, default false)
```

Comment:
@legiangthanh Have you confirmed similar option names of other plugins (especially OSS plugins under https://github.com/embulk/)?  We should have consistent names with other plugins as far as possible.

* I know there already are some variations, but I have never seen `tls_enable`.
* `ftp` and JDBC plugins use just `ssl`.
* "SSL" is deprecated in general, so `tls` could be reasonable.
* `ftp` has `ssl_verify`, which seems similar (but opposite) to `tls_insecure`?

Reply:
@dmikurube , 
```
I know there already are some variations, but I have never seen tls_enable.
ftp and JDBC plugins use just ssl.
"SSL" is deprecated in general, so tls could be reasonable.
```
Sorry, I don't notice option names of other plugins, I will change it to ssl.

`ftp has ssl_verify, which seems similar (but opposite) to tls_insecure?`
This option is similar option supported by mongo command, please refer https://www.mongodb.com/docs/manual/reference/connection-string/#mongodb-urioption-urioption.tlsInsecure

Reply:
As I mentioned, `ssl` -> `tls` could be reasonable in this age. But `_enable` has not been common for other plugins.

> This option is similar option supported by mongo command, please refer
> https://www.mongodb.com/docs/manual/reference/connection-string/#mongodb-urioption-urioption.tlsInsecure

Got it... that is a point of design choice. Then `tls_insecure` sounds reasonable, but nice to explain it ("the same with `mongo` command") both in `README` and `PluginType`.